Header header
int64 prompt_id
string type
string text
string[] options
float32 display_duration

